gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/utils/APICoreUtil.bal

```diff
@@ -61,7 +73,7 @@ function getAPIs () (json) {
     json apiList;
     try {
         http:ClientConnector client = create http:ClientConnector(getAPICoreURL());
-        string query = "?labels=Private,Public";
+        string query = "?labels=Default";
```

Comment:
We have to get this values from environment properties when ballerina docker get start